.npmrc

```diff
@@ -2,3 +2,4 @@ auto-install-peers=true
 package-manager-strict=false
 manage-package-manager-versions=true
 engine-strict=true
+registry=https://registry.npmjs.org/
```

Comment:
We can't git ignore this, since this file is being tracked, hence the need to commit this.